test/powershell/Modules/Microsoft.PowerShell.Utility/Test-Json.Tests.ps1

```diff
@@ -0,0 +1,104 @@
+Describe "Test-Json" -Tags "CI" {
+    BeforeAll {
+        $validSchemaJson = @"
+            {
+            'description': 'A person',
+            'type': 'object',
+            'properties': {
+                'name': {'type': 'string'},
+                'hobbies': {
+                'type': 'array',
+                'items': {'type': 'string'}
+                }
+                }
+            }
+"@
+
+        $invalidSchemaJson = @"
+            {
+            'description',
+            'type': 'object',
+            'properties': {
+                'name': {'type': 'string'},
+                'hobbies': {
+                'type': 'array',
+                'items': {'type': 'string'}
+                }
+                }
+            }
+"@
+
+        $validJson = @"
+            {
+                'name': 'James',
+                'hobbies': ['.NET', 'Blogging', 'Reading', 'Xbox', 'LOLCATS']
+            }
+"@
+
+        $invalidTypeInJson = @"
+            {
+                'name': 123,
+                'hobbies': ['.NET', 'Blogging', 'Reading', 'Xbox', 'LOLCATS']
+            }
+"@
+
+        $invalidTypeInJson2 = @"
+            {
+                'name': 123,
+                'hobbies': [456, 'Blogging', 'Reading', 'Xbox', 'LOLCATS']
+            }
+"@
+
+        $invalidNodeInJson = @"
+            {
+                'name': 'James',
+                'hobbies': ['.NET', 'Blogging', 'Reading', 'Xbox', 'LOLCATS']
+                errorNode
+            }
+"@
+}
+
+    It "Json is valid" {
```

Comment:
First 4 It can be converted to one with testcases and splatting.

Reply:
I tried to do it, but there are too many parameters, and these tests are losing visibility. Maybe leave as is?